classes/autoptimizeCriticalCSSCore.php

```diff
@@ -203,7 +206,12 @@ public function ao_ccss_extend_types() {
         }
 
         // Templates.
-        $templates = wp_get_theme()->get_page_templates();
+        // Transient cache to avoid frequent disk reads
+        $templates = get_transient('ao_ccss_page_tpls');
+        if(!$templates) {
+            $templates = wp_get_theme()->get_page_templates();
+            set_transient('ao_ccss_page_tpls', $templates, HOUR_IN_SECONDS);
```

Comment:
minor: maybe name it `autoptimize_ccss_page_templates`, that's more in line of other transient usage in AO :)

Reply:
Sure, changed.